app/models/repair.rb

```diff
@@ -1,8 +1,15 @@
 class Repair < ActiveRecord::Base
-  attr_accessible :category, :comments, :description, :item_id, :person_id, :service_date, :status, :user_id
+  attr_accessible :category, :comments, :description,
+                  :item_id, :person_id, :service_date,
+                  :status, :user_id, :cost
   belongs_to :item
   belongs_to :person
-  
+  validates_numericality_of :cost
+
   STATUS_CHOICES = ['Needed', 'In-progress', 'Awaiting Parts', 'Completed - Repaired', 'Completed - No Trouble Found']
   CATEGORY_CHOICES = ['Repair', 'Inspection' ]
+
+  def repairer_name
```

Comment:
I don't think simply tying to people is going to work. Sometimes repairs will be made in house, and that will work, but  other times we'll send it out to a garage.